src/index.js

```diff
@@ -78,15 +79,19 @@ function IdTokenVerifier(parameters) {
  * - if token is not expired and valid (if the `nbf` claim is in the past)
  *
  * @method verify
- * @param {String} token id_token to verify
- * @param {String} [nonce] nonce value that should match the one in the id_token claims
+ * @param {string} token id_token to verify
+ * @param {string} [requestedNonce] nonce value that should match the one in the id_token claims
  * @param {verifyCallback} cb callback used to notify the results of the validation
  */
 IdTokenVerifier.prototype.verify = function(token, requestedNonce, cb) {
+  if (!cb && requestedNonce && typeof requestedNonce == 'function') {
+    cb = requestedNonce;
+    requestedNonce = undefined;
+  }
```

Comment:
In reality this would be better off as an options object we can pass in as this looks messy and won't scale if we add other things here. No fault of your own, that's just the way it is. Let me come back to you on this to let you know if that's a path we can go down, as it's a breaking change.

Reply:
On reflection, let's keep it as is. It's a breaking change but not one I feel warrants a new major version in itself.